source/expressions.tex

```diff
@@ -3289,7 +3289,7 @@
 \tcode{T}''.
 
 \item Otherwise, if \tcode{E1.E2} refers to a non-static member
```

Comment:
How about:

> Otherwise, if `E1.E2` refers to a non-static member function of type `T`, then `E1.E2` is a prvalue of type `T` designating that non-static member function. The expression can be used only as the left-hand operand of a member function call ([class.mfct]). [Note: ... ]

(And similarly in the static member function case.)

Do we really need to say "if `E1.E2` refers to a non-static member function then `E1.E2` designates that non-static member function"? What distinction between "refers to" and "designates" are we trying to create here?

After simplifying this wording, the bullets for static and non-static member functions are the same other than the "can only be used" sentence and its note; perhaps the common wording could be extracted?

Reply:
Maybe what we want to say is that E2 names a non-static member function of type T, in which case E1.E2 is a prvalue of type T designating that non-static member function. We make a connection from the individual parts to the compound "E1.E2" expression.

Reply:
static member functions are lvalues, non-static member functions are prvalues, it seems

Reply:
Updated; should be better now.